src/middlewared/middlewared/plugins/system_advanced/config.py

```diff
@@ -317,7 +317,7 @@ async def sed_global_password(self):
         ))['sed_passwd']
         return passwd if passwd else await self.middleware.call('kmip.sed_global_password')
 
-    @accepts()
+    @accepts(roles=['SYSTEM_ADVANCED_READ'])
```

Comment:
You don't need the roles. since you're setting `@no_auth_required`. I was just making sure we'd validated it worked properly when not authenticated.

Perhaps add an API test as an unauthenticated session and verify you can call it properly.

Reply:
When making this call as a RO admin, even if `@no_authz_required` is included, without the roles a RO will still get `Not authorized`. Adding the role and just having `@no_auth_required` seems to be sufficient. 

Unless you're saying that a RO admin not being allowed is intended

Reply:
no_auth_required and no_authz_required should bypass privilege checks always. If that is not happening then a NAS ticket should be opened.